Java/karate/pom.xml

```
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.deque</groupId>
+  <artifactId>axe-devtools-java-examples-karate</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>axe DevTools API Integration with Karate Framework</name>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+	</properties>
+	
+	<repositories>
+        <repository>
+            <id>deque</id>
+            <url>https://agora.dequecloud.com/artifactory/devtools-maven/</url>
+        </repository>
+    </repositories>
+
+	<dependencies>
+	
+        <dependency>
+	      <groupId>com.deque.html.axe-devtools</groupId>
+	      <artifactId>selenium</artifactId>
+	      <version>4.2.2</version>
+	    </dependency>
```

Should use the latest version here - 
```suggestion
        <dependency>
	      <groupId>com.deque.html.axe-devtools</groupId>
	      <artifactId>selenium</artifactId>
	      <version>4.6.0</version>
	    </dependency>
```